superx/models/__init__.py

```diff
@@ -1,14 +1,22 @@
 from sqlalchemy import Integer, Column, Text, Boolean, Float
-
+from flask_login import LoginManager, UserMixin
 from app import db
 
 
-class User(db.Model):
+class User(UserMixin, db.Model):
     __tablename__ = 'user'
 
     id = Column(Integer, primary_key=True)
```

Comment:
is this an auto_increment or are you planning on typing in the id?

Reply:
it should be auto_increment automatically no?

Reply:
Probably,